static/css/tabzilla/tabzilla.css

```diff
@@ -0,0 +1,34 @@
+#tabzilla {
```

Comment:
This file is taken verbatim from the tabzilla repo, but with a few changes I'll highlight